### ISSUE TITLE
onwindowresize event handler unbind

### ISSUE THREAD

**Issue by asliceofpizza007** [Flyer53/jsPanel4#136]
Hi, I'm currently using your library in my project. It really meets my needs. Thanks for that.

Here's the question：
When I set `onwindowresize: true`  in my panel's config, it seems it won't unbind the window resize event handler even the panel is closed which might cause the memory leaking.And because of this,  it may cause the error below while you close the panel which is set as maximize.And If I set onwindowresize as a function, it would not trigger the function whether the panel is set as maximize or minimize.(same issue occurred on jspanel official site ( [https://jspanel.de/#options/onwindowresize](url) ) )

`Uncaught TypeError: Cannot read property 'clientWidth' of null
    at HTMLDivElement.i.maximize (jspanel.min.js:1)
    at jspanel.min.js:1`

The version of jspanel I'm using is v4.10.2.I would very appreciate if you could fix this !

**Comment by Flyer53** [comment]
Hi there,
I'm currently out of town for a few days. So I'll look at this issue when I'm back home next week.
Thanks for your report.

**Comment by Flyer53** [comment]
Ok, I could reproduce your issue. It seems to happen only when the panel is maximized before you close it. Correct?
A workaround that seems to work is using an `onbeforeclose` handler that normalizes the panel before it's closed:
```javascript
jsPanel.create({
    onwindowresize: true,
    maximizedMargin: 10,
    syncMargins: true,
    onbeforeclose: function(panel){
      panel.normalize();
      return true; // important! otherwise the panel won't close
    }
});
```
At least it works in my dev environment using Firefox and Chrome.
Please let me know whether it works in your case as well.

**Comment by Flyer53** [comment]
@asliceofpizza007 
I just uploaded a modified [v4.11.0-beta](https://jspanel.de/downloads/jspanel-4.11.0-beta.zip) which should fix the issue. Please give it a try and let me know how it works out.

**Comment by asliceofpizza007** [comment]
Hi, @Flyer53 
I'm so sorry that I didn't check my mail lately. I was totally forgot. I'd apologize for that.
And I have tried both of ways that you're offering. They work !
The v4.11.0-beta version also unbinds the window resize event that would be a great help for preventing the memory leaking.

I do appreciate your help for fixing this problem.

**Comment by Flyer53** [comment]
Fixed with v4.11.0 release
